acceptance/api/v1/orgs_test.go

```diff
@@ -198,5 +201,68 @@ var _ = Describe("Namespaces API Application Endpoints", func() {
 				env.VerifyOrgNotExist(org)
 			})
 		})
+
+		Describe("GET api/v1/namespacematches", func() {
+			It("lists all namespaces for empty prefix", func() {
+				response, err := env.Curl("GET", fmt.Sprintf("%s/api/v1/namespacematches", serverURL),
+					strings.NewReader(""))
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response).ToNot(BeNil())
+				defer response.Body.Close()
+				bodyBytes, err := ioutil.ReadAll(response.Body)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response.StatusCode).To(Equal(http.StatusOK), string(bodyBytes))
+
+				var namespaces []string
+				err = json.Unmarshal(bodyBytes, &namespaces)
+				Expect(err).ToNot(HaveOccurred())
+
+				// See global BeforeEach for where this namespace is set up.
+				Expect(namespaces).Should(ContainElements(org))
```

Comment:
If I understand it correctly we are only checking for the existence of one namespace. Maybe we should use a list of namespaces for testing for this test case just to be sure that it returns that?

Reply:
The namespace name we are checking for (`org`) is the only namespace which is truly known at this point. If there are others I do not know of them. Especially in light of testing against an existing cluster which may may already have whatever other namespaces the CI is not aware of.